src/workbookManager.ts

```
@@ -19,8 +19,10 @@ export class WorkbookManager {
             templateFile === undefined
                 ? await JSZip.loadAsync(WorkbookTemplate.SIMPLE_QUERY_WORKBOOK_TEMPLATE, { base64: true })
                 : await JSZip.loadAsync(templateFile);
-
-        return await this.generateSingleQueryWorkbookFromZip(zip, query, docProps, tableData);
+        const initialData = 
```

Instead of fixing it on behalf of caller - fail explicitly here.